showQueueDuration.js

```diff
@@ -9,22 +9,24 @@
 // are added to Spotify. This is to ensure compatibility
 // with older Spotify versions
 const QUEUE_H1_CLASSES = [
-  ".main-type-canon",
-  ".DG9CsoFIptJhAneKoo_F",
-  ".queue-queuePage-header",
+	".main-type-canon",
+	".DG9CsoFIptJhAneKoo_F",
+	".queue-queuePage-header",
+	".mP6tR7IgjiamGqpBW5ai"
 ].join(", ");
 
 (function ShowQueueDuration() {
+	const queueHeader = document.querySelector(QUEUE_H1_CLASSES);
+
 	// Only run on the queue page
-	if (Spicetify.Platform?.History.location.pathname !== "/queue") {
+	if (!queueHeader) {
```

Comment:
There are other pages that have HTML elements with classes `.main-type-cannon` and so on.
If we remove this check, the "— HH:mm:ss" will appear in these pages too.

I'd keep it as it is (stay in a loop checking every second if we're in the queue page).
We can add `|| !queueHeader` to the check though. It would prevent the script from crashing if it doesn't find a 'Queue" header 👍

Reply:
Yeah I figured, I had only done that because for some reason it kept throwing errors for me when I ran it as a custom extension, even though it had no problems as an official extension.